requirements.txt

```diff
@@ -0,0 +1,2 @@
+pygame
```

Comment:
Ok perfect! I actually wasn't sure about which one to add. In situations where (I feel like) my code should work on at least the latest version, should I just add that? Otherwise, is it the best solution to test previous versions (up to when it makes sense to check them) and go with the first one that gives some problems?

Reply:
You dont need to test previous version, usually you should work with the latest version (apart from situations in which you need an older version) and specify that in the requirements!

Ideally in a well maintained codebase you will release new versions of your code updated with new versions of your dependencies 😃